language/move-prover/src/prelude.bpl

```diff
@@ -72,7 +72,7 @@ function {:constructor} $IntegerType() : $TypeValue;
 function {:constructor} $AddressType() : $TypeValue;
 function {:constructor} $StrType() : $TypeValue;
 function {:constructor} $VectorType(t: $TypeValue) : $TypeValue;
-function {:constructor} $StructType(name: $TypeName, ps: $TypeValueArray, ts: $TypeValueArray) : $TypeValue;
+function {:constructor} $StructType(name: $TypeName, ts: $TypeValueArray) : $TypeValue;
```

Comment:
I remember you explained ps and ts to me once.  But I can't recall any more.  Since we only have ts, I assume this is simply the type arguments of the generic type which matches my intuition which is great.

Reply:
We should remove ts in the near future. It are the types of the fields, but we don't need them if we do not do dynamic type assumptions.

Reply:
The other array are the field types. We would need it if we do dynamic type assumption, we can add it back later once we need this.